packages/slice-machine/lib/builders/CustomTypeBuilder/Layout/Header/primaryButton.tsx

```diff
@@ -51,7 +51,9 @@ export const PrimaryButton: React.FC<PrimaryButtonProps> = ({
     [
       CustomTypeStatus.New,
       CustomTypeStatus.Modified,
-      CustomTypeStatus.Unknown,
+      CustomTypeStatus.UnknownDefault,
```

Comment:
judging by the previus code we don't need this one

Reply:
I think the only condition needs is
```suggestion
     customTypeStatus != CustomTypeStatus.synced
```